### PR TITLE
Implement override optmizers arguments

### PR DESCRIPTION
**Pull request by ysugimoto**
This PR will be enable to override each optimizers command arguments from https://github.com/ysugimoto/aws-lambda-image/issues/134.

I had planed that adding `optimizers` configuration key, and if exists, override command arguments.
However, we should care about each optimizers have to work with stdin/stdout mode, so I complement argument as much as possible.

### CODE REVIEW

File: lib/optimizer/Mozjpeg.js [hunk 1]
```diff
@@ -9,12 +9,13 @@ class Mozjpeg extends Optimizer {
      * @constructor
      * @extends Optimizer
      * @param Number|undefined quality
+     * @param Array|undefined args
      */
-    constructor(quality) {
+    constructor(quality, args) {
```
Comment (1000ch) [inline lib/optimizer/Mozjpeg.js]: If target Node.js is 6.x,

```js
constructor(quality, args = ["-optimize", "-progressive"]) {
```

File: lib/optimizer/Pngquant.js [hunk 2]
```diff
@@ -9,12 +9,17 @@ class Pngquant extends Optimizer {
      *
      * @constructor
      * @extends Optimizer
+     * @param Array|undefined args
      */
-    constructor() {
+    constructor(args) {
```
Comment (1000ch) [inline lib/optimizer/Pngquant.js]: ditto

File: bin/configtest [hunk 3]
```diff
@@ -112,6 +121,11 @@ var reset   = '\u001b[0m';
         process.exit(1);
     }
 
+    function formatArray(ary) {
+        ary = ary.map(function(v) { return "\"" + v + "\""; });
+        return "[" + ary.join(", ") + "]";
```
Comment (1000ch) [inline bin/configtest]:
```js
return `[${ary.map(v => `"${v}"`).join(", ")}]`; 
```

**Comment by ysugimoto** [comment]
@1000ch We should support node v4.3.x because probably there are some user who using v4.3.x old version.

**Pull request merged**